### PR TITLE
Add support for events.

### PR DESCRIPTION
**Pull request by geeknoid**
Events are used for special announcements. There are stickers and banners that can be
displayed to the user. These can be used to announce an imminent release with a
sticker and countdown clock, or can be used to invite users to a future
conference, or can be used to announce that a new release is available for download.

See the authoring guide for instructions on how to use these announcements.

### CODE REVIEW

File: content/en/events/index.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (frankbu) [inline content/en/events/index.md]: What it is this for?

File: content/en/events/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+then this event is considered to have happened at the release date of the latest Istio release. This is used to
+automatically display a banner that says "Istio X.Y.Z has just been released" whenever a new release is produced.
+
+- event_date. The date of the event in YYYY-MM-DD format.
+
+- countdown_days. How long before the event to display the item to the user.
+
+- event_days. How long after the event to display the item to the user.
+
+- visibility. How many times to show the content to the user, or the string `always` to display it on every page during
```
Comment (frankbu) [inline content/en/events/README.md]: I wonder if a boolean would be sufficient ... display once or always. Picking a number in between would seem hard, and might end up looking like random behavior. Anyway, if it's not hard to keep track, i guess it doesn't hurt to allow any number.

File: content/en/events/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+the event's period. A value of 3 would mean that the first three pages visited by the user during the period will display
+the content, and the content will be hidden on subsequent page loads.
+
+- timeout. The amount of time the content is visible to the user on a given page. After that much time passes, the item will be removed from
```
Comment (frankbu) [inline content/en/events/README.md]: Maybe this is overkill too, unless it's easy to support?

File: content/en/events/index.md (hunk 1 above)
Comment (geeknoid) [reply content/en/events/index.md]: Ignorable infra stuff. This just tells Hugo that this whole tree is not to be rendered as normal pages. There's the same kind of file at the top of the boilerplates directory for the same reason.

File: content/en/events/README.md (hunk 3 above)
Comment (geeknoid) [reply content/en/events/README.md]: That's really easy to support, but also really frivolous...

**Pull request merged**